pom.xml

```diff
@@ -45,9 +47,22 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>step-2020-johndallard</deploy.projectId>
+          <deploy.projectId>step-2020-halvin</deploy.projectId>
```

Comment:
Can you comment this one?

Reply:
Fixed

Reply:
I meant to comment it out. I thought you didn't want "step-2020-halvin" be in the github?

Reply:
You're right! I was being silly - fixed!